src/styles/styles.css

```diff
@@ -0,0 +1,178 @@
+html {
+  font-family: Roboto, Arial, 'Helvetica Neue', 'Segoe UI', sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+p {
+  margin: 0;
+}
+
+.card {
+  display: flex;
+  width: 198px;
+  height: 406px;
+}
```

Comment:
The width and height for `.card` are hardcoded which is not a flexible solution. Consider using max-width or percentages for width and padding-top for aspect ratio if needed.